liiatools/cans_pipeline/spec/__init__.py

```diff
@@ -58,3 +58,25 @@ def load_schema() -> DataSchema:
 
     # Now we can parse the full schema into a DataSchema object from the dict
     return DataSchema(**full_schema)
+
+
+@lru_cache
+def load_summary_sheet_mapping() -> DataSchema:
```

Comment:
Really minor but I don't think this is actually returning a DataSchema, this could either just be removed or changed to dict. Same for below